### DISCUSSION TITLE
How do we bypass/autoGrant the permission for IOS app in between the application

### DISCUSSION THREAD

**Discussion by vishal14121996** [appium/appium#20724] [General]
How do we bypass/autoGrant the permission for IOS app in between the application:

So in my app when i'm launch the applicaiton it ask the permission for notification, that will be autoAccept by providing the Appium:autoGrantPermission='true'
but after login into the app in my application there is one page on which after clicking on 'confirm' button it will ask the Location,camera permission..

How can i bypass it?

My ios app is build on flutter technology.

Adding the screenshot of before & after clicking on confirm button.
<img width="451" alt="Screenshot 2024-11-04 at 5 37 25 PM" src="https://github.com/user-attachments/assets/2c92d3d0-d515-4261-9a7e-ed9159d38a59">
<img width="451" alt="Screenshot 2024-11-04 at 5 37 34 PM" src="https://github.com/user-attachments/assets/318e9b2b-a2f9-4991-b15b-9ce60bb9a432">
<img width="447" alt="Screenshot 2024-11-04 at 5 37 44 PM" src="https://github.com/user-attachments/assets/b88c037f-703e-4f47-a84c-203a043a208c">

**Comment by KazuCocoa** [comment]
`acceptAlertButtonSelector` in https://appium.github.io/appium-xcuitest-driver/latest/reference/settings/ would help to configure the acceptance logic
